python_2/hw_2_5.py

```diff
@@ -0,0 +1,14 @@
+# 5) Есть строка со словами и числами, разделенными пробелами (один пробел между словами и/или числами).
+# Слова состоят только из букв. Вам нужно проверить есть ли в исходной строке три слова подряд.
+# Для примера, в строке "hello 1 one two three 15 world" есть три слова подряд.
+
+import re
+
+string_1 = 'hello 1 one two three 15 world'
```

Comment:
will not work -  'hello 1 one two three four 15 world'
will not work -  'hello 1 one two three four five 15 world' ...